CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### citus v6.0.1 (November 29, 2016) ###
+
+* Fixes a bug preventing DML queries while creating colocated table
+
+* Fixes a bug causing incorrectly passing down NULL parameters as text
+
+* Fixes a bug causing failures during pg_upgrade
```

Comment:
Move this to the top (most impactful or important `CHANGELOG` should go to the top).